app/v2/controllers/ErrorTranslator.scala

```diff
@@ -42,6 +42,14 @@ trait ErrorTranslator {
     def convert(input: E): PresentationError
   }
 
+  val jsonToXmlValidationErrorConverter = new Converter[FailedToValidateError] {
```

Comment:
We should probably get a set of unit tests for this as this is a new converter. It should suffice to copy the Validation Error block in the tests:

https://github.com/hmrc/common-transit-convention-traders/blob/a71421e279960fe95f525c9910ce0c8a48c2e706/test/v2/controllers/ErrorTranslatorSpec.scala#L64-L105

and then just ensure the XML validation error test is changed to return an Internal Server Error.